components/FeedbackView/ButtonGroup.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { JSX } from 'react';
+
+interface Variant {
+  title: string;
+  subtitle: string;
+  id: string;
+}
+
+interface ButtonGroupProps {
+  variants: Variant[];
+  handleFeedbackSubmit: (feedback: string) => void;
+}
+
+/**
+ * Component that renders a group of buttons.
+ * @param {object} props - The component props
+ * @param {object[]} props.variants - The list of button variants to render
+ * @param {Function} props.handleFeedbackSubmit - The function to handle the feedback submit
```

Comment:
refactor (blocking): use the proper types

```suggestion
 * @param {Variant[]} props.variants - The list of button variants to render
 * @param {(feedback: string) => void;} props.handleFeedbackSubmit - The function to handle the feedback submit
```

You can also create a type for the feedback callback and use that in both places, e.g.

```typescript
type FeedbackCallback = (feedback: string) => void;
```